src/basilisp/lang/obj.py

```diff
@@ -105,7 +135,7 @@ def entry_reprs():
     if print_meta and meta:
         return f"^{lrepr(meta, **kwargs)} {start}{seq_lrepr}{end}"
 
-    return f"{start}{seq_lrepr}{end}"
+    return f"{'#:'+ns_same if ns_same else ''}{start}{seq_lrepr}{end}"
```

Comment:
```suggestion
    return f"{('#:'+ns_same) if ns_same else ''}{start}{seq_lrepr}{end}"
```

This looks slightly less ambiguous to me.

Reply:
Updated